toptal/picasso
# fix: update Lexical to 0.11.2

[ER-33576](https://toptal-core.atlassian.net/browse/ER-33576)
[FX-4168](https://toptal-core.atlassian.net/browse/FX-4168)

### Description

Updates Lexical editor to fix some of the existing issues such as:

- unwanted line breaks when copy/pasting HTML
- exceptions such as ER-33576, caused by trying to format line breaks into headings

### How to test

- [Temploy](https://picasso.toptal.net/ER-33567-update-lexical) 
- Open a Google Doc document with one paragraph and one level 3 heading directly under it ([example](https://docs.google.com/document/d/1ypKFwoP36ZtqYKrlRqtYPJo27Rz-agIEZChRk53KIrc/edit?usp=sharing))
- Copy and paste everything into RichTextEditor story
- Move cursor to the beginning of H3
- Press backspace and then select all text
- Convert text to heading, there should be no error
 

### Development checks



> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>


[ER-33576]: https://toptal-core.atlassian.net/browse/ER-33576?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[FX-4168]: https://toptal-core.atlassian.net/browse/FX-4168?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
